climada/entity/exposures/litpop/litpop.py

```diff
@@ -0,0 +1,1108 @@
+        if shape is None:
+            self.set_countries(countries, res_km=None, res_arcsec=res_arcsec,
+                               exponents=(exponent, 0), fin_mode='none', 
+                               reference_year=reference_year, gpw_version=None,
+                               data_dir=data_dir, resample_first=True)
+        elif countries is not None:
+            LOGGER.info("Using shape provided instead of countries' shapes. "
+                        "To use countries' shapes, set shape=None")
+        if shape is not None:
+            self.set_custom_shape(shape, res_arcsec=res_arcsec, exponents=(exponent,0),
+                                  fin_mode='none', in_countries=countries,
+                                  reference_year=reference_year,
+                                  gpw_version=None, data_dir=data_dir,
+                                  resample_first=True)
+
+    def set_pop(self, countries=None, shape=None, res_arcsec=30, exponent=1,
+                reference_year=2020, gpw_version=None, data_dir=None):
+        """
+        initiate Exposure with pure population data
+
+        Parameters
+        ----------
+        countries : list or str (optional)
+            list containing country identifiers (name or iso3)
+        shape : Shape, Polygon or MultiPolygon, optional
+            shape to crop results to, overrules countries
+        res_arcsec : int, optional
+            Resolution in arc seconds. The default is 30.
+        exponent : int or float >= 0, optional
+            exponent n in Pop^n. The default is 1.
+        reference_year : int, optional
+            Target year, closest GPW data year is returned.
+            The default is 2020.
+        gpw_version : int, optional
+            specify GPW data verison. The default is 11.
+        data_dir : Path, optional
+            data directory. The default is None.
+
+        Raises
+        ------
+        ValueError
+            Either countries or shape is required.
+        """
+
+        if countries is None and shape is None:
+            raise ValueError("Either countries or shape required. Aborting.")
+        if shape is None:
+            self.set_countries(countries, res_km=None, res_arcsec=res_arcsec,
+                               exponents=(0, exponent), fin_mode='pop', 
+                               reference_year=reference_year, gpw_version=gpw_version,
+                               data_dir=data_dir)
+        elif countries is not None:
+            LOGGER.info("Using shape provided instead of countries' shapes. "
+                        "To use countries' shapes, set shape=None")
+        if shape is not None:
+            self.set_custom_shape(shape, res_arcsec=res_arcsec, exponents=(0,1),
+                                  fin_mode='pop', in_countries=countries,
+                                  reference_year=reference_year,
+                                  gpw_version=gpw_version, data_dir=data_dir)
+
```

Comment:
```suggestion
        if shape is not None:
            if countries is not None:
                LOGGER.info("Using shape provided instead of countries' shapes. "
                        "To use countries' shapes, set shape=None")
            self.set_custom_shape(shape, res_arcsec=res_arcsec, exponents=(0,1),
                                  fin_mode='pop', in_countries=countries,
                                  reference_year=reference_year,
                                  gpw_version=gpw_version, data_dir=data_dir)

```